SuiteCareers/SuiteCareers/Program.cs

```diff
@@ -5,13 +5,14 @@
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
-var connectionString = builder.Configuration.GetConnectionString("DefaultConnection") ?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");
-builder.Services.AddDbContext<ApplicationDbContext>(options =>
+var connectionString = builder.Configuration.GetConnectionString("SuiteCareers-Mac-Sqlite") ?? throw new InvalidOperationException("Connection string 'DefaultConnection' not found.");
```

Comment:
Are we going to be switching back and forth between SQL lite and sqlserver, is there a way around this.